packages/components/package.json

```
@@ -0,0 +1,85 @@
+{
+  "name": "@web3-ui/components",
+  "license": "MIT",
+  "version": "1.0.0",
+  "private": false,
+  "engines": {
+    "node": ">=16.0.0",
+    "yarn": "^1.5"
+  },
+  "description": "React Typescript Library Boilerplate",
+  "author": "Dhaiwat Pandya",
```

should this be Developer DAO? Not sure what you think @Dhaiwat10 
```suggestion
  "author": "Developer DAO",
```